src/main/scheduler/scheduler.go

```diff
@@ -72,11 +72,19 @@ func (s *Scheduler) notifyOrder(order model.Order, transactionIds []string) erro
 		transactionIds,
 	)
 
-	return (*s.notifier).Send(s.config.NotifyEmailAddress, notification.Subject(), notification.Body())
+	var errors []error
+	for _, notifier := range s.notifiers {
```

Comment:
Could we move this to a function? Otherwise it's duplicated in every notification function